app/javascript/react/container/AirlineShowContainer.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { browserHistory, Link } from 'react-router';
+import AirlineShowTile  from '../components/AirlineShowTile'
+
+class AirlineShowContainer extends Component {
+ constructor(props) {
+   super(props);
+   this.state= {
+     airline: "",
+     destinations: "",
+     rating: ""
+
+   }
+ }
+ componentDidMount(){
+
+ let id = this.props.params.id
+ console.log(id)
```

Comment:
Please remove this `console.log` -- debugging lines shouldn't be committed into your codebase because they're not important for its functioning